pkg/build/builder/cmd/scmauth/scmauth.go

```diff
@@ -0,0 +1,25 @@
+package scmauth
```

Comment:
something inside me is telling me that should package should live elsewhere ("pkg/util/scm" ?)